internal/npm_install/test/package/delete_build_files.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @fileoverview
+ * BUILD files in node modules have to be deleted because runfiles are not used
```

Comment:
Ahhhh. That makes sense and solves the mystery 👍

Reply:
This is special to the test fixture because node_modules here are installed *outside* of the test, then added to the runfiles.